### ISSUE TITLE
Jupyter "directory not found" is back

### ISSUE THREAD

**Issue by GitHK** [ITISFoundation/osparc-simcore#2405]
As seen in the screenshot, reported by @newton1985, the issue is back again.

![Screenshot 2021-06-29 at 14 13 01](https://user-images.githubusercontent.com/5694077/123795262-1f0cae80-d8e4-11eb-8cb3-3ab40a14d0fb.png)

Logs from the service

[jupyter-smash_300ad71d-039f-4b4b-b83f-05a0d871a7ed.1.u8wlqszm19m2cgf4pz5zwkwsc.log](https://github.com/ITISFoundation/osparc-simcore/files/6733286/jupyter-smash_300ad71d-039f-4b4b-b83f-05a0d871a7ed.1.u8wlqszm19m2cgf4pz5zwkwsc.log)

I will remind everyone that this is a generic error displayed by the jupyter.

Previous closed issue https://github.com/ITISFoundation/osparc-simcore/issues/2017

**Comment by pcrespov** [comment]
Some inputs from last [fix](https://github.com/ITISFoundation/osparc-issues/issues/412#issuecomment-823135143)

**Comment by pcrespov** [comment]
@msteiner
So I encountered again the issue where my ML PyTorch crashes which happened to me several times already. As I was told to contact you right away when it happens again, I will try to document what happened. Sorry in advance if I am overseeing something or making mistakes in how to use osparc.
So before starting the training of the neural network everything was working fine. Then somewhere in the 2nd epoch the execution failed telling me a file which was there before is not there anymore. When closing the error message indeed the file seems to have been deleted somehow. I tried then to reestablish everything and run again but then it gave me an error when trying to reinstall all the required packages. I send u some screenshots I took of the different stages. Hope it is clear. Thanks a lot in advance for your help.
